include/arch/arm/cortex_m/scripts/linker.ld

```diff
@@ -528,6 +512,31 @@ SECTIONS
 	KEEP(*(.gnu.attributes))
 	}
 
+#if defined CONFIG_ARM_FIRMWARE_HAS_SECURE_ENTRY_FUNCS
+#if CONFIG_ARM_NSC_REGION_BASE_ADDRESS != 0
+	#define NSC_ALIGN . = CONFIG_ARM_NSC_REGION_BASE_ADDRESS
```

Comment:
@oyvindronningstad here, if the user defines CONFIG_ARM_NSC_REGION_BASE_ADDRESS to something non-zero, we do not check for the nRF IDAU alignment requirements.

Reply:
Sorry, yes, I was referring to the rounding in the SPM code.